shell/packages/sandstorm-accounts-ui/LICENSE

```diff
@@ -0,0 +1,20 @@
+forked from Meteor's accounts-ui from https://github.com/meteor/meteor/tree/ab9ef40d258bbb9b9de453600495c8337577c565/packages/accounts-ui-unstyled
```

Comment:
Sorry, I guess when you asked if I wanted this in multiple changes I wasn't really thinking about the fact that you were bringing in Meteor code. It would be nice to be able to see a diff between the original meteor code and your code. Could we submit the code verbatim first, then update it as a second change?